argfile.konflux

```diff
@@ -4,4 +4,4 @@ LIBSODIUM_VERSION=1.0.20
 VLLM_TGIS_ADAPTER_VERSION=0.6.3
 FLASHINFER_VERSION=https://github.com/flashinfer-ai/flashinfer/releases/download/v0.2.1.post1/flashinfer_python-0.2.1.post1+cu124torch2.5-cp38-abi3-linux_x86_64.whl
 VLLM_WHEEL_VERSION=0.7.2
-VLLM_WHEEL_INDEX=https://gitlab.com/api/v4/projects/66664052/packages/pypi/simple
+VLLM_WHEEL_INDEX=https://gitlab.com/api/v4/projects/68349498/packages/pypi/simple
```

Comment:
I just tried installing vLLM from this index url, but looking at it, it does not contain the correct version:

```bash
python -c "from vllm import __version__; print(__version__)"
```
shows `0.7.2.0` instead of `0.7.2.1`.

Thinking it's just a versioning issue (i.e. version file not updated), I checked `site-packages/vllm` for the transcription endpoint (`grep -R transcriptions site-packages/vllm`), but it's not there: